charts/README.md

```diff
@@ -0,0 +1,31 @@
+Deploy CSI Baremetal to kind-cluster using operator
+---------------------
+
+Last update: 26.03.2021
+
+! Kind-cluster has pre-loaded csi images
+
+1. Set CSI version
+
+```
+export csiVersion=0.0.13-375.3c20841
+```
+
+2. Build operator image and load to kind
+    
+```
+make docker-build
+make kind load
+```
+
+3. Deploy operator
+
+```
+helm install csi-baremetal-operator ./charts/csi-baremetal-operator/
+```
+
+4. Deploy csi-baremetal
+
+```
+helm install csi-baremetal ./charts/csi-baremetal-deployment/ --set image.tag=${csiVersion} --set image.pullPolicy=IfNotPresent --set driver.drivemgr.image.name=csi-baremetal-loopbackmgr --set driver.drivemgr.deployConfig=true --set scheduler.patcher.enable=true
```

Comment:
Can we have short name for drive manager as it was before - `loopbackmgr`, `basemgr`, etc.?

Reply:
done